### ISSUE TITLE
Question: pretrained model necessary in training.yaml?

### ISSUE THREAD

**Issue by bramiozo** [AnswerDotAI/ModernBERT#195]
I don't quite understand why I have to fill in the name of pre-trained model in the pretrain .yaml, it does not start from scratch?

**Comment by NohTow** [comment]
Hello,
It is used to read some default values if these are not set (e.g, num layers, embedding dim, ...).
At some point we should remove that entirely, but this is how MosaicBERT (the base of this repository) was working.
